business-api-ecosystem/values/values-keyrock.yml

```diff
@@ -26,6 +26,21 @@ route:
 ingress:
   ## should there be an ingress to connect keyrock with the public internet
   enabled: false
+  ## anotations to be added to the ingress
+  annotations:
+    # annotation for microk8s using the built-in ingress addon
+    kubernetes.io/ingress.class: public
```

Comment:
Thank you for your contribution. The annotations are specific to the K8s setup. Therefore I would suggest to comment this out and state that this could be used for microk8s.